flink-python/pom.xml

```diff
@@ -334,6 +334,7 @@ under the License.
 									<excludes>
 										<exclude>org/apache/beam/vendor/bytebuddy/**</exclude>
 										<exclude>org/apache/beam/repackaged/core/org/antlr/**</exclude>
+										<exclude>META-INF/maven/org.antlr/antlr4-runtime/**</exclude>
```

Comment:
Should we simply exclude `META-INF/maven/org.antlr`?

Reply:
Yes, I'll do that.